File: TPraticas/Aula11.a/codigofonte/LOverflow2.java

import java.util.Scanner;
public class LOverflow2
{
  public static void main(String[] args)
  {
    int[] tests = new int[10];
    int test;
    int count;
    Scanner scan = new Scanner(System.in);

    System.out.print("Quantos números? ");
    count = scan.nextInt();
    for (int i =0 ; i < count; i++)
    {
    	System.out.print("Introduza número: ");
   	test  = scan.nextInt();
    	tests[i]= test;
    }
  }
}

File: TPraticas/Aula11.a/codigofonte/LOverflow2.cpp


#include <iostream>

using namespace std;

int main(void) {
    int test = 1;
    int  tests[10];
    int num_elems;

   cout << "Quantos números? ";
   cin >> num_elems;
   
    for (int i = 0; i < num_elems; i++) {  
      cout << "Insira número: ";
      cin >> test;
      tests[i] = test;
    }

    return 0;
}


